src/components/Activities/ActivitiesPage.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const ActivitiesPage = () => {
```

Comment:
I suggest renaming the `components` folder to `pages` because it contains mostly pages.
Then create another `components` folder to store `Drawer`, `NavigationBar`, `PostItem`, ...

Reply:
got it